docs/build.sbt

```diff
@@ -0,0 +1,10 @@
+// Uses the out of the box generic theme.
+paradoxTheme := Some(builtinParadoxTheme("generic"))
+
+scalaVersion := "2.12.2"
+
+paradoxProperties in Compile ++= Map(
+  "snip.g8root.base_dir" -> "../../../../src/main/g8",
+  "snip.g8src.base_dir" -> "../../../../src/main/g8/src/main/",
+  "snip.g8srctest.base_dir" -> "../../../../src/main/g8/src/test/"
```

Comment:
It looks the most recent version of sbt-giter8 has a setting for this:

https://github.com/foundweekends/giter8/blob/0.8.x/plugin/src/main/scala/giter8/Giter8Plugin.scala#L52

Although maybe it's `src/g8` instead of `src/main/g8` now?